src/dali/transaction_resolver.py

```diff
@@ -142,7 +142,8 @@ def _update_spot_price_from_web(transaction: AbstractTransaction, global_configu
         )
         if conversion.rate is None:
             raise Exception(
-                f"Spot price for {transaction.timestamp_value}:{transaction.asset}->{global_configuration[Keyword.NATIVE_FIAT.value]}"
+                f"Spot price for {transaction.unique_id + ':' if transaction.unique_id else ''}"
+                f"{transaction.timestamp_value}{transaction.asset}->{global_configuration[Keyword.NATIVE_FIAT.value]}"
```

Comment:
Missing `:` after `{transaction.timestamp_value}` (it's needed to separate fields)

Reply:
Done!